pages/getting-started/manage-git-repositories/workspace-encryption.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Workspace Encryption Guide
+description: How to use `plural crypto backups` to create, list, and restore workspace backups.
+---
+
+Out of the box, Plural encrypts all configuration in your Plural workspace and stores an encryption key on your local machine.
+This means that if anyone gets a copy of your Plural Git repository/workspace, all sensitive information will appear encrypted to them.
+
+If you are changing machines or collaborating with colleagues, you'll need to know how to create and restore from the workspace
+encryption backups.
+
+## Your encryption key
+
+When you initialize your Plural repository, Plural creates a directory in your home director called `.plural`. Within this directory
+is a file called `key`, which is your local encryption key. This key is required to decrypt any sensitive configuration within your
+Plural workspace.
+
+## Encryption backup operations
+
+### Create workspace backup
+
+To create a backup for your local `key`, run `plural crypto backups create`. In the case that you lose your local encryption key,
```

Comment:
can we put these commands into code blocks?  It adds the copy/paste button

Reply:
yep, done